### ISSUE TITLE
Conditional Variables broken in progressive succession configurations?

### ISSUE THREAD

**Issue by dlincoln** [Desertsnowman/Caldera-Forms#458]
I made a quote generator form with conditional show statements that progressively open as the user clicks selections. It’s live here: http://baleties.com/product/steel-bandingstrapping/

The first disclosure/show statement works fine - click either option to reveal the selecter containing the next possible choices. But after that it stops working - everything in each of the revealed selectors is linked to options farther down that should show after you select the an option in the dropdowns revealed before. Those elements stay hidden.

I’m attaching the form below - something is amiss in CF, or the way I’m using it.
[steel-banding-quote-export.json.zip](https://github.com/Desertsnowman/Caldera-Forms/files/169715/steel-banding-quote-export.json.zip)

**Comment by dlincoln** [comment]
Anyone got an update on this? User error or bug?

**Comment by dlincoln** [comment]
I'm leaning more towards bug, or design flaw. Some additional context in recent testing narrows things a bit further:
Conditional variables that depend on the state of a dropdown selector's settings seem to be related to the object being displayed. For example, I can hide/show another dropdown selector with a condition in a dropdown selector fairly reliably. However, I cannot hide/show an html content block or a populated radio button from a dropdown selector's changed-state.

**Comment by dlincoln** [comment]
e.g. on this page/form http://baleties.com/product/steel-bandingstrapping-copy/ the first option (none) reveals another dropdown selector. Each of the other options is conditionally tied to either a radio button or text display, but none do anything. All conditions are set to "or", not "and".

**Comment by dlincoln** [comment]
More data. I made a dummy form that progressively disclosed fields, and it seems to work perfectly in very complex, overlapping contexts:
http://baleties.com/?page_id=115&preview=true&cf_preview=CF56f16a4baa1dd

So the basic code seems fine.

I then started to rebuild the form from scratch, and the new form broke when I attached a conditional to reveal a text box vs a radio button... it took a minute but I discovered the bug:

The dropdown selecter that was failing contained " characters in each line:
e.g. 1/2" x .015
When I remove the " the form works fine!

SUMMARY: **There IS a bug, and it's that labels and values in dropdown selectors (maybe other places?) are not being escaped before they're parsed for comparison in conditional contexts.**

Is there a workaround for this or must I live without inch marks in a table of measurements?
